### ISSUE TITLE
Can I share data stored using ngrx/db between 2 different tabs in the same browser? if yes how?

### ISSUE THREAD

**Issue by daBishMan** [ngrx/platform#495]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
</code></pre>

## What is the current behavior?

We have an ng4, ngrx 4 application, we have a need when the user clicks on profile we need to open the profile in a new tab.  I am not sure if I can share the data or how.  

Basically I want to maintain the state between the 2 or more tabs, I am open to using ngrx/db as long as I can keep it as long as those tabs are open.

## Expected behavior:

It would be great if I can do that.

**Comment by agborkowski** [comment]
https://github.com/ngrx/platform/blob/master/example-app/README.md

"This application utilizes @ngrx/db to persist the collection across sessions; @ngrx/store to manage the state of the app and to cache requests made to the Google Books API; @angular/router to manage "

**Comment by dinvlad** [comment]
Yes you can through localStorage, using https://github.com/btroncone/ngrx-store-localstorage/issues/40#issuecomment-336283880 for example. DB would probably be a bit more appropriate for that, but I'm not sure how well-supported it is at the moment (unlike localStorage).

**Comment by daBishMan** [comment]
Thank you @agborkowski  and @dinvlad for your feedback.

@dinvlad I will take a closer look at your code in the comment, thank you.
